### PR TITLE
Add OCM tourney

### PR DESCRIPTION
**Pull request by Metak94**
This is the OMC tourney wikia page. 
There is all the information, and futher  matchs, inscriptions, schedules, etc.

### CODE REVIEW

File: wiki/Tournaments/OCM/2020/en.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
+---
+tags:
+- OCM 2020
+- OCM2020
+---
+# Osu Cup Master (OCM): Osu!catch edition
```
Comment (WalterToro) [inline wiki/Tournaments/OCM/2020/en.md]: No creo que los acronimos sean necesarios en el titulo, y para mantener la criteria, mejor dejar el osu en minusculas

```suggestion
# osu! Cup Master 3: osu!catch edition
```

File: wiki/Tournaments/OCM/2020/en.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+<!--
+->Inserte logo aca<-
+-->
+
+Welcome everyone to the 3rd version  of the Osu Cup Master. This will be the osu!catch version. We hope you'll enjoy it and have fun on this competition.
```
Comment (WalterToro) [inline wiki/Tournaments/OCM/2020/en.md]: Para que siga el formato del resto de los articulos

```suggestion
The **osu! Cup Master 3: osu!catch edition** (***OCM3***) is a a two-on-two team based osu!catch tournament hosted by ![][flag_PA] [Rei-](https://osu.ppy.sh/users/2317102) and ![][flag_MX] [daxeroz](https://osu.ppy.sh/users/1170156). It is the 3rd installment of the osu! Cup Master.
```

File: wiki/Tournaments/OCM/2020/en.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2019-12-14 / 2020-01-25 |
+| Live Drawings | 2020-02-01 |
+| Group Stage | 2020-02-08 / 2020-02-09 |
+| Round of 16 | 2020-02-15 / 2020-02-16 |
+| Quarterfinals | 2020-02 22 / 2020-02 23 |
```
Comment (WalterToro) [inline wiki/Tournaments/OCM/2020/en.md]:
```suggestion
| Quarterfinals | 2020-02-22 / 2020-02-23 |
```

File: wiki/Tournaments/OCM/2020/en.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+| Semifinals | 2020-02-29 / 2020-03-01 |
+| Finals | 2020-03-07 |
+| Grand Finals | 2020-03-14 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :--- |
+| ![][GCrown] "1st place" | winner badge for each member (to be confirmed) |
```
Comment (WalterToro) [inline wiki/Tournaments/OCM/2020/en.md]:
```suggestion
| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Profile badge (to be confirmed) |
```

File: wiki/Tournaments/OCM/2020/en.md [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+| ![][SCrown] "2nd place" | a month of osu!supporter for each member (to be confirmed) |
```
Comment (WalterToro) [inline wiki/Tournaments/OCM/2020/en.md]:
```suggestion
| ![Silver Crown](/wiki/shared/GCrown.png "2nd place") | 1 month of osu!supporter tag |
```

File: wiki/Tournaments/OCM/2020/en.md [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+<!--
+->inserte chapita aca<-
+-->
+
+## Organisation
+
+The OCM tournament is run by the next comunity members:
```
Comment (WalterToro) [inline wiki/Tournaments/OCM/2020/en.md]:
```suggestion
The osu! Cup Master 3: osu!catch edition Tournament is run by various osu! community members.
```

File: wiki/Tournaments/OCM/2020/en.md [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+
+| Position | Members |
+| :-: | :--- |
+| Host | ![][flag_PA] [Rei-](https://osu.ppy.sh/users/2317102) , ![][flag_MX] [daxeroz](https://osu.ppy.sh/users/1170156) |
```
Comment (WalterToro) [inline wiki/Tournaments/OCM/2020/en.md]:
```suggestion
| Host | ![][flag_PA] [Rei-](https://osu.ppy.sh/users/2317102), ![][flag_MX] [daxeroz](https://osu.ppy.sh/users/1170156) |
```

File: wiki/Tournaments/OCM/2020/en.md [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+<!--
+->inserte chapita aca<-
+-->
+
+## Organisation
+
+The OCM tournament is run by the next comunity members:
+
+| Position | Members |
+| :-: | :--- |
```
Comment (WalterToro) [inline wiki/Tournaments/OCM/2020/en.md]:
```suggestion
| :-: | :-- |
```

File: wiki/Tournaments/OCM/2020/en.md [hunk 9]
```diff
@@ -0,0 +1,163 @@
...
+| Host | ![][flag_PA] [Rei-](https://osu.ppy.sh/users/2317102) , ![][flag_MX] [daxeroz](https://osu.ppy.sh/users/1170156) |
+| Manager | ![][flag_CL] [Metak](https://osu.ppy.sh/users/2861255) |
+| Map Selector | ![][flag_MX] [daxeroz](https://osu.ppy.sh/users/1170156) |
+| Commentator | ![][flag_PE] [-Boltic-](https://osu.ppy.sh/users/5297904) , ![][flag_PA] [Rei-](https://osu.ppy.sh/users/2317102) |
```
Comment (WalterToro) [inline wiki/Tournaments/OCM/2020/en.md]:
```suggestion
| Commentator | ![][flag_PE] [-Boltic-](https://osu.ppy.sh/users/5297904), ![][flag_PA] [Rei-](https://osu.ppy.sh/users/2317102) |
```

File: wiki/Tournaments/OCM/2020/en.md [hunk 10]
```diff
@@ -0,0 +1,163 @@
...
+| Referee | ![][flag_CL] [Hinoshi](https://osu.ppy.sh/users/9200197) ,  ![][flag_PE] [-Boltic-](https://osu.ppy.sh/users/5297904) , ![][flag_AR] [jerem1as](https://osu.ppy.sh/users/6117864) , ![][flag_MX] [MILK_16](https://osu.ppy.sh/users/9137627) , ![][flag_UY] [Neimex23](https://osu.ppy.sh/users/7293703) , ![][flag_GB] [StartAndSelect](https://osu.ppy.sh/users/8720555/osu) |
```
Comment (WalterToro) [inline wiki/Tournaments/OCM/2020/en.md]:
```suggestion
| Referee | ![][flag_CL] [Hinoshi](https://osu.ppy.sh/users/9200197), ![][flag_PE] [-Boltic-](https://osu.ppy.sh/users/5297904), ![][flag_AR] [jerem1as](https://osu.ppy.sh/users/6117864), ![][flag_MX] [MILK\_16](https://osu.ppy.sh/users/9137627), ![][flag_UY] [Neimex23](https://osu.ppy.sh/users/7293703), ![][flag_GB] [StartAndSelect](https://osu.ppy.sh/users/8720555/osu) |
```

File: wiki/Tournaments/OCM/2020/en.md [hunk 11]
```diff
@@ -0,0 +1,163 @@
...
+| Designer | ![][flag_MX] [ShadowMerry](https://osu.ppy.sh/users/3104864) |
+| Streamer | ![][flag_MX] [daxeroz](https://osu.ppy.sh/users/1170156) |
+
+
```
Comment (WalterToro) [inline wiki/Tournaments/OCM/2020/en.md]:
```suggestion
```

File: wiki/Tournaments/OCM/2020/en.md (hunk 3 above)
Comment (Metak94) [reply wiki/Tournaments/OCM/2020/en.md]: si ese error lo notamos y lo corregimos en el foro.

File: wiki/Tournaments/OCM/2020/en.md (hunk 1 above)
Comment (Metak94) [reply wiki/Tournaments/OCM/2020/en.md]: okas toda la razon.

File: wiki/Tournaments/OCM/2020/en.md (hunk 2 above)
Comment (Metak94) [reply wiki/Tournaments/OCM/2020/en.md]: ya okas, una descripcion general mas que un bienvenidos.

**Comment by cl8n** [comment]
the logo is currently an empty file

**Comment by Metak94** [comment]
yes i know xD. when i submitted this pull, the logo was a work in progress issue. I'm asking if it ready to put on the page.

**Comment by cl8n** [comment]
at just 40px wide I'm not sure all of those team images are going to be readable. were you planning to use them somewhere else too? I don't see the point of adding large(r) images if they'll all be scaled down anyway

**Comment by Metak94** [comment]
We ask team images to put them on the osu!lazer for the stream. The original images can be seen on this link (https://docs.google.com/spreadsheets/d/e/2PACX-1vTyCEwFRpnYATXmv5EPlM6En76Yc00-YMW8RJYQX46pORKBa0oQMbw24M9RA0gR2y5j3HDwqCrHmo6W/pubhtml#) 
However, we decide to make them flag sized images to use them on the tournament script, and the staff told me that they want smaller size on the wiki page to make it look tidier. 
I personally agree with you, the images are not readable, do you think its better leave the images at the original size?

**Comment by TPGPL** [comment]
Unless they have a special meaning, i don't think there's a point in keeping the team images in the article.

If you don't mind removing them, i will do the changes locally so you don't have to do it through github web interface

**Comment by cl8n** [comment]
wdym "special meaning", metak explained what they are for in the last comment

(forgot to post this here, I only said on discord) I was playing around with the sizes earlier and found that bounding them to about 80x80 seems reasonable, you can still read most of the text and it's not obnoxious to scroll through

**Comment by TPGPL** [comment]
By special meaning i mean that if they were removed, would that be crucial for the article or would that make no difference at all

**Comment by Metak94** [comment]
The team images are not crucial, they are just a customization that we ask to the teams to do (to put in the stream). If you guys think this is unnecessary on the wiki article, then ill erase it. If is not, i will set the size of the images to 80px wide as clayton suggested.
And im going to update the tournament information (pools, results, etc), we are on the third week after all xD

**Comment by Metak94** [comment]
i leave the team images at 80 px wide like clayton suggested. If you wiki friends consider this is unnecesary just erase it,  i understand the page must be as clean and tidy as posible.

**Comment by TPGPL** [comment]
tracking PR in #3841

**Pull request closed without merging**